src/components/FilterSearch.tsx

```diff
@@ -93,7 +93,7 @@ export function FilterSearch({
   const hasResults = sections.flatMap(s => s.results).length > 0;
   const [currentFilter, setCurrentFilter] = useState<Filter>();
 
-  const handleSelectDropdown = useCallback((_value, _index, itemData) => {
+  const handleFilterDropdown = useCallback((itemData, select) => {
```

Comment:
can this be renamed? right now it sounds like this function should handle what happens after some "filtering" behavior is performed on the dropdown, which doesn't make too much sense

Reply:
that's true, what would you suggest? `handleDropdown` seemed too vague but it's being used for both `onSelect` and `onToggle` in this component.